msal/application.py

```diff
@@ -673,6 +674,16 @@ def initiate_auth_code_flow(
             If the elapsed time is greater than this value,
             Microsoft identity platform will actively re-authenticate the End-User.
 
+        :param str response_mode:
+            OPTIONAL. Specifies the method with which response parameters should be returned.
+            For security purposes, the value should be form_post. In this mode, response parameters
```

Comment:
The current wording "For security purposes, the value should be form_post" could be perceived as the default behavior is insecure, but a web app powered by MSAL Python does not obtain tokens via query parameter thus is insusceptible to the "[query mode concern](https://openid.net/specs/oauth-v2-multiple-response-types-1_0.html#Security)" in the first place.

How about this?
```suggestion
            The default value is equivalent to ``query``, which is still secure enough in MSAL Python
            (because MSAL Python does not transfer tokens via query parameter in the first place).
            For even better security, we recommend using the value ``form_post``.
            In "form_post" mode, response parameters
```